tests/models/bart/test_modeling_bart.py

```diff
@@ -504,6 +504,40 @@ def test_inputs_embeds(self):
             with torch.no_grad():
                 model(**inputs)[0]
 
+    # BartForSequenceClassification does not support inputs_embeds
```

Comment:
In this case, where the model raises a `NotImplementedError`, we can add a try/catch in the test, skipping the test if this exception is caught